create_containers.sh

```diff
@@ -0,0 +1,51 @@
+#!/usr/bin/env bash
+# shellcheck disable=SC2154,SC2143,SC2034,SC1091
+set -euo pipefail
+
+source ./docker_functions.sh
+
+# Create networks for containers
+log "Checking for existing networks..."
+for network in "${networks[@]}"
+do
+  if [ "$(docker network ls | grep -w "${network}")" ]; then
+    log "Network ${network} exists, skipping..."
+  else
+    OUTPUT=$(networks)
+    log "Network ${network} created"
+  fi
+done
+
+# Create volumes for containers
+log "Checking for existing volumes..."
+for volume in "${volumes[@]}"
+do
+  if [ "$(docker volume ls -q | grep -w "${volume}")" ]; then
+    log "Volume ${volume} exists, skipping..."
+  else
+    OUTPUT=$(volumes)
+    log "Volume ${volume} created"
+  fi
+done
+
+# Spawn todo list node.js app
+if [ "$(docker ps -a | grep -w "${app}")" ]; then
+  # cleanup
+  log "Existing ${app} container found - nuking & re-spawning it"
+  docker rm -f "${app}" > /dev/null
```

Comment:
In most cases, we want the output available unless there is a specific reason or bug requiring that it is hidden.